### PR TITLE
Define blood vessel dataset and fix filename bug

### PR DESCRIPTION
**Pull request by yamengxi**

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -37,4 +37,4 @@ repos:
     rev: v1.3.1
     hooks:
       - id: docformatter
-        args: ["--in-place", "--wrap-descriptions", "79"]
+        args: ["--in-place", "--wrap-descriptions", "79"]
```
Comment (yamengxi) [inline .pre-commit-config.yaml:40]: I didn't modify any code here.

**Comment by hellock** [comment]
Task linked: [CU-4vmywh 添加医疗数据集CHASE_DB1, HRF, DRIVE, STARE的定义](https://app.clickup.com/t/4vmywh)

File: mmseg/datasets/drive.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import os.path as osp
+
+from .builder import DATASETS
+from .custom import CustomDataset
+
+
+@DATASETS.register_module()
+class DriveDataset(CustomDataset):
```
Comment (xvjiarui) [inline mmseg/datasets/drive.py:8]:
```suggestion
class DRIVEDataset(CustomDataset):
```

File: mmseg/datasets/hrf.py [hunk 3]
```diff
@@ -0,0 +1,27 @@
+import os.path as osp
+
+from .builder import DATASETS
+from .custom import CustomDataset
+
+
+@DATASETS.register_module()
+class HrfDataset(CustomDataset):
```
Comment (xvjiarui) [inline mmseg/datasets/hrf.py:8]:
```suggestion
class HRFDataset(CustomDataset):
```

File: mmseg/datasets/stare.py [hunk 4]
```diff
@@ -0,0 +1,27 @@
+import os.path as osp
+
+from .builder import DATASETS
+from .custom import CustomDataset
+
+
+@DATASETS.register_module()
+class StareDataset(CustomDataset):
```
Comment (xvjiarui) [inline mmseg/datasets/stare.py:8]:
```suggestion
class STAREDataset(CustomDataset):
```

File: mmseg/datasets/chase_db1.py [hunk 5]
```diff
@@ -0,0 +1,27 @@
+import os.path as osp
+
+from .builder import DATASETS
+from .custom import CustomDataset
+
+
+@DATASETS.register_module()
+class Chase_db1Dataset(CustomDataset):
```
Comment (xvjiarui) [inline mmseg/datasets/chase_db1.py:8]:
```suggestion
class ChaseDB1Dataset(CustomDataset):
```

**Comment by yamengxi** [comment]
Waiting for review!

**Review by xvjiarui — APPROVED** [review]

**Pull request merged**